app/javascript/components/GuitarInfo.js

```diff
@@ -7,10 +8,68 @@ import {
   Card,
   CardTitle,
   CardBody,
-  CardFooter
+  CardFooter,
+  Button,
+  Modal,
+  ModalVariant,
+  Form,
+  FormGroup,
+  TextInput
 } from '@patternfly/react-core';
+import { guitarUrl } from './constants.js';
+
+const GuitarInfo = ({ id, name, url, price, description }) => {
+  const [isFormModalOpen, toggleFormModal] = useReducer(
+    (isFormModalOpen) => !isFormModalOpen,
+    false
+  );
+
+  const initialFormValues = {
+    name: '',
+    url: '',
+    price: null,
+    description: ''
+  };
+
+  // initialFormValues according to the button pressed!! have 1 for create and 1 for edit
+
+  // Form Attributes useStates():             Initial Values
+  const [nameValue, setNameValue] = useState(initialFormValues['name']);
+  const [ulrValue, setUrlValue] = useState(initialFormValues['url']);
+  const [priceValue, setPriceValue] = useState(initialFormValues['price']);
+  const [descriptionValue, setDescriptionValue] = useState(initialFormValues['description']);
+
+  const editGuitar = () => {
+    // Q: Do I need to axios.get(id) the edited guitar from the DB? I think so, because the form values are empty !
+
+    // Populate fields of form with the edited guitar's attributes:
+    setNameValue(name);
+    setUrlValue(url);
+    setPriceValue(price);
+    setDescriptionValue(description);
```

Comment:
since all of those parameters are available from props, can we use them as the initial value in the `useState`s?